src/services/ReactNativeLogger.ts

```diff
@@ -159,54 +159,55 @@ export default class ReactNativeLogger {
     const logFilePath = this.getReactNativeLogsFilePath()
     console.debug('React Native logs will be piped to ' + logFilePath)
 
-    const oldDebug = console.debug
-    const oldLog = console.log
-    const oldInfo = console.info
+    const consoleFns: { [key: string]: (message?: any, ...optionalParams: any[]) => void } = {
+      debug: console.debug,
+      log: console.log,
+      info: console.info,
+      // console.error displays a red box, therefore, we console.info instead
+      error: console.info,
+      // console.warn displays a yellow box, therefore, we console.info instead
+      warn: console.info,
```

Comment:
This is something I noted a long time ago,
We can disable the Red and Yellow boxes and keep the actual level in the log message. Which is better I think.

But I'm also thinking it would be beneficial to actually let the default Red and Yellow boxes pop up.
Forcing us to act on these errors during development, instead of waiting to see them in Sentry.

We can discuss further and make the explicit change in a dedicated PR.